k-distribution/tests/regression-new/issue-1803-paramProds/Makefile

```diff
@@ -0,0 +1,3 @@
+KOMPILE_BACKEND?=haskell
+
+include ../../../include/kframework/ktest-fail.mak
```

Comment:
I don't understand this test. You're importing ktest-fail, so it's supposed to not kompile, but the output file doesn't show any error. That seems like a bug to me. We should never output nothing if compilation fails.

Reply:
I copied the files from another test. I wanted to check that it kompiles correctly, with no errors or warnings.